src/main/java/seedu/address/logic/commands/student/StudentDeleteCommand.java

```diff
@@ -20,36 +22,44 @@ public class StudentDeleteCommand extends Command {
     public static final String COMMAND_WORD = "studentDelete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the student identified by the index number used in the displayed person list.\n"
+            + ": Deletes the student identified by the index number (separated by whitespace)"
```

Comment:
```suggestion
            + ": Deletes the student identified by the index number(s) (separated by whitespace)"
```

This would better describe how the user can specify one or more than one index number.